uber/jinja.py

```diff
@@ -0,0 +1,61 @@
+from uber.common import *
+
+
+class JinjaEnv:
+    _env = None
+    _exportable_functions = {}
+    _filter_functions = {}
+    _template_dirs = []
+
+    @staticmethod
+    def append_template_dir(dirname):
+        """
+        Add another template directory we should search when looking up templates.
+        """
+        JinjaEnv._template_dirs.insert(0, dirname)
+
+    @staticmethod
+    def env():
```

Comment:
Just be aware that ``@on_startup`` tasks do NOT happen when running ``sep`` commands.  So if you have anything that should always happen even for ``sep`` commands, don't put it in an ``@on_startup`` function.